lib/ansible/modules/windows/win_get_url.py

```diff
@@ -73,6 +73,19 @@
     type: bool
     default: yes
     version_added: '2.4'
+  checksum:
```

Comment:
Seems like separate checksum_algorithm and checksum args would preferable here (even though that's not the way get_url currently does it)- it probably should.

Reply:
We make it similar to get_url module.

Reply:
Done